### PR TITLE
Add custom-columns and custom-template-file to --help text

### LINKED ISSUE #28314: kubectl: jsonpath is not very intuitive
As a user, I want to see a list of (for example) pods and their imageIDs.

There are some hacks.

I think I could use jq to get one columns:  `kubectl get pods -ojson | jq .items[].status.containerStatuses[].imageID`

I think I could learn jsonpath: kubectl get pods -o jsonpath='{.items[*].status.containerStatuses[].imageID}'

But tying this back to a particular pod is hard.

I would really like to do something more like:  `kubectl get pods --columns name, imageID`

### PR DESCRIPTION
**Pull request by dims**
Fixes #28314

### CODE REVIEW

File: pkg/kubectl/cmd/get.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func NewCmdGet(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:     "get [(-o|--output=)json|yaml|wide|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...] (TYPE [NAME | -l label] | TYPE/NAME ...) [flags]",
+		Use:     "get [(-o|--output=)json|yaml|wide|custom-template=...|custom-template-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...] (TYPE [NAME | -l label] | TYPE/NAME ...) [flags]",
```
Comment (deads2k) [inline pkg/kubectl/cmd/get.go]: looks like the flag enum is typoed `custom-columns-file` and `custom-columns` right?

Comment (dims) [inline pkg/kubectl/cmd/get.go]: @deads2k oops! thanks.

**Comment by dims** [comment]
@k8s-bot test this #IGNORE

**Comment by dims** [comment]
@k8s-bot test this issue: #IGNORE

**Comment by dims** [comment]
@k8s-bot test this issue: #IGNORE

**Comment by dims** [comment]
@deads2k David, this PR seems to have passed all the tests. Can you please take a quick look?

**Comment by deads2k** [comment]
> @deads2k David, this PR seems to have passed all the tests. Can you please take a quick look?

sorry, bot removed the tag on a rebase i guess.

**Pull request merged**